src/portals/hope/features/tasks/list-items/TaskListItem.tsx

```diff
@@ -208,55 +238,121 @@ export const TaskListItem: React.FC<
       remindKeys={[Keys.Up, Keys.Down]}
     >
       <ListItem
-        whileHover={{ scale: 1.02 }}
-        whileTap={{ scale: 0.98 }}
         key={task.id}
         onClick={onClick}
         selected={selected}
+        onMouseEnter={() => setShowCta(true)}
+        onMouseLeave={() => setShowCta(false)}
       >
-        <Spacing left="tiny">
-          <Flex gap="medium" align="center">
-            <Popover
-              contents={
-                task.assignedTo
-                  ? convertEmailToName(task.assignedTo)
-                  : 'Unassigned'
-              }
-              position="right"
+        <Flex gap="medium" align="center">
+          <Popover
+            contents={
+              task.assignedTo
+                ? convertEmailToName(task.assignedTo)
+                : 'Unassigned'
+            }
+            position="right"
+          >
+            <UserCircle
+              assignedToMe={assignedToMe}
+              unassigned={task.assignedTo === null}
             >
-              <UserCircle
-                assignedToMe={assignedToMe}
-                unassigned={task.assignedTo === null}
-              >
-                {task.assignedTo
-                  ? convertEmailToSignature(task.assignedTo)
-                  : ''}
-              </UserCircle>
-            </Popover>
-            <MemberFlag>{flag}</MemberFlag>
-            <Name
-              available={!!title}
-              onClick={(e) => {
-                onClickTitle?.()
-                e.stopPropagation()
-              }}
-            >
-              {title ? title : 'Not available'}
-            </Name>
-          </Flex>
-        </Spacing>
-        <span className="preview">{task.description}</span>
-        <Flex align="center">
+              {task.assignedTo ? convertEmailToSignature(task.assignedTo) : ''}
+            </UserCircle>
+          </Popover>
+          <MemberFlag>{flag}</MemberFlag>
```

Comment:
Maybe we should swap to this? Where flag can be any of the Flags available in `constants.ts`
```suggestion
          <MemberFlag flag={flag} />
```